crates/core/tedge/src/cli/certificate/remove.rs

```diff
@@ -18,16 +19,25 @@ impl Command for RemoveCertCmd {
     }
 
     fn execute(&self) -> anyhow::Result<()> {
-        self.remove_certificate()?;
+        match self.remove_certificate()? {
+            RemoveCertResult::Removed => println!("Certificate was successfully removed."),
+            RemoveCertResult::NotFound => println!("There is no certificate to remove."),
```

Comment:
The punctation does not provide much value, so I would recommend to remove it.

```suggestion
            RemoveCertResult::Removed => println!("Certificate was successfully removed"),
            RemoveCertResult::NotFound => println!("There is no certificate to remove"),
```

Reply:
Though a general comment, it is better to write user output to stderr instead of stdout.

Reply:
> Though a general comment, it is better to write user output to stderr instead of stdout.

I was under the impression that stderr is just for errors. I can imagine the use case in which as you say, user output is better in stderr, is when a user pipes stdout to a file, that way they would still see user messages in the terminal, in which case it really does make more sense and stderr makes for some unfortunate naming.

Are there any other cases or perhaps could you point me somewhere where I can read more about that?

Reply:
Here are a few links
* https://julienharbulot.com/python-cli-streams.html
* https://stackoverflow.com/questions/7977852/what-are-the-conventions-for-stdout-stderr-messages

But it is all about making command work well with other tooling, especially when piping the output to other tools like grep, and usually user facing messages are not very interesting to pipe to other commands.